article/app/model/LiveBlogHelpers.scala

```diff
@@ -79,4 +82,38 @@ object LiveBlogHelpers {
     }.getOrElse(Right(NotFound))
 
   }
+
+  def blockTextJson(page: LiveBlogPage, number: Int): JsValue = {
+
+    case class TextBlock(
```

Comment:
It's unusual to define a model with json implicits nested in a function

Reply:
Good shout for that, I'd like to leave that for a follow up pr though I've done too much for one round and this is existing code that's been moved.